docs/cloud/integrations/looker.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Connect to Steampipe Cloud from Looker
+sidebar_label: Looker
+---
+
+##  Connect to Steampipe Cloud from Looker
+
+[Looker](https://cloud.google.com/looker) is an enterprise platform for BI, data applications, and embedded analytics that helps you explore and share insights in real time. It connects to many databases, including Postgres, and enables users to explore, query, and visualize data.
+
+Steampipe provides a single interface to all your cloud, code, logs and more. Because it's built on Postgres, Steampipe provides an endpoint that any Postgres-compatible client -- including Looker -- can connect to.
+
+The [Connect](/docs/cloud/integrations/overview) tab for your workspace provides the details you need to connect Looker to Steampipe Cloud.
+
+<div style={{"marginBottom":"2em","borderWidth":"thin", "borderStyle":"solid", "borderColor":"lightgray", "padding":"20px", "width":"90%"}}>
+<img src="/images/docs/cloud/steampipe-cloud-connect-details.jpg" />
+</div>
+
+##  Connect to Steampipe CLI from Looker
+
+You can also connect Looker to [Steampipe CLI](https://steampipe.io/downloads). To do that, run `steampipe service start --show-password` and use the displayed connection details.
+
+```
+Steampipe service is running:
+
+Database:
+
+  Host(s):            localhost, 127.0.0.1, 192.168.29.204
+  Port:               9193
+  Database:           steampipe
+  User:               steampipe
+  Password:           99**_****_**8c
+  Connection string:  postgres://steampipe:99**_****_**8c@localhost:9193/steampipe
+  ```
+
+## Getting started
+
+Looker is accessible through the GCP console. Here we will create a Steampipe Cloud connection from Looker.
+
+To create a new connection, click on `Database` then `Connections` from the admin page. Then click `Add Connection` and select `PostgreSQL 9.5+` from the `Dialect` dropdown list, enter the Steampipe Cloud connection details. Set the SSL Mode to on form advanced settings and click `Test` to `Verify` then click `Save`.
```

Comment:
```suggestion
To create a new connection, click on `Database` then `Connections` from the admin page. Then click `Add Connection`,  select `PostgreSQL 9.5+` from the `Dialect` dropdown list, and enter the Steampipe Cloud connection details. In `Additional Settings`, set the SSL Mode to `On`.  Click `Test` to `Verify`, then click `Save`.
```

Reply:
I assume SSL Mode is under Additional Settings? Maybe expand to show that?

Reply:
I've updated the Image as well that shows the SSL slider.